init_sd_card.scripts/init_sd_card.sh

```diff
@@ -324,13 +326,19 @@ services:
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
   http-server:
-    image: resin/raspberrypi3-alpine-python:slim
```

Comment:
what was this one doing ?

Reply:
this is the base image for `rpi-simple-server`.

We prefer `rpi-simple-server` because then we can update it if we want.